unitpackage/remote.py

```diff
@@ -44,7 +44,7 @@ def collect_zipfile_from_url(url):
 
 ECHEMDB_DATABASE_URL = os.environ.get(
     "ECHEMDB_DATABASE_URL",
-    "https://github.com/echemdb/website/archive/refs/heads/gh-pages.zip",
+    "https://github.com/echemdb/electrochemistry-data/releases/download/0.1.0-alpha.2/data-0.1.0-alpha.2.zip",
```

Comment:
@saraedum, upon each release of the database on https://github.com/echemdb/electrochemistry-data, we would like to update this link, where `0.1.0-alpha.2` is replaced with the new tag. Is there a simple approach with a bot, that can create a pull request?

Reply:
I fixed the problem in a different way. The URL is now provided by a module in the `electrochemistry-data` module, which can be installed. I guess this is apparent from the changes made.